### ISSUE TITLE
dotnet35 does not install successfully

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noahocker/winetricks#181]
```
What steps will reproduce the problem?
1. install wine 1.4rc6 on OS X
2. make a new prefix
3. run "winetricks dotnet35"

What is the expected output? What do you see instead?
According to the Wine AppDB, .Net 3.5 should install successfully. Instead, 
installation stops with the message "Download complete. You can now disconnect 
from the Internet." The progress bar is full, but the installation never 
completes.

What version of the product are you using? On what operating system?
Wine 1.4rc6, latest winetricks (r781 at this time), OS X 10.7.

Please provide any additional information below.
The last lines in the terminal are:

fixme:heap:HeapSetInformation 0x0 1 0x0 0
err:ole:CoGetClassObject class {a9e69610-b80d-11d0-b9b9-00a0c922e750} not 
registered
err:ole:create_server class {a9e69610-b80d-11d0-b9b9-00a0c922e750} not 
registered
fixme:ole:CoGetClassObject CLSCTX_REMOTE_SERVER not supported
err:ole:CoGetClassObject no class object {a9e69610-b80d-11d0-b9b9-00a0c922e750} 
could be created for context 0x15
fixme:msi:ACTION_CustomAction msidbCustomActionTypeTSAware not handled


```

Original issue reported on code.google.com by `camillo....@gmail.com` on 3 Mar 2012 at 8:06

**Comment by GoogleCodeExporter** [comment]
```
I can reproduce this bug in Arch Linux, wine 1.5, on a clean WINEPREFIX. I 
attach a log with the messages of the terminal related to dotnet35 installation.
```

Original comment by `gaul...@gmail.com` on 7 Apr 2012 at 11:33

Attachments:
- [dotnet35.log](https://storage.googleapis.com/google-code-attachments/winetricks/issue-181/comment-1/dotnet35.log)

**Comment by GoogleCodeExporter** [comment]
```
Same problem, exactly same last lines, happens with:
"winetricks dotnet35" and "winetricks -q dotnet35", clean wine prefix.
OS: OS X 10.7
winetricks version: 20120308
wine version: 1.5.3

In my case dotnet30 and dotnet30sp1 installs and works fine.
Also, in "wine control", in Add/Remove Programs it lists .NET 3.0
```

Original comment by `ignas2...@gmail.com` on 30 Apr 2012 at 6:32

**Comment by GoogleCodeExporter** [comment]
```
http://bugs.winehq.org/show_bug.cgi?id=25060#c16 gives a possible recipe that 
could be incorporated into 'winetricks dotnet35'.  Please try it out and let me 
know if it works for you.
```

Original comment by `daniel.r...@gmail.com` on 25 May 2012 at 8:36

**Comment by GoogleCodeExporter** [comment]
```
It hangs while doing 'winetricks -q dotnet35'.
```

Original comment by `camillo....@gmail.com` on 29 May 2012 at 5:50

**Comment by GoogleCodeExporter** [comment]
```
I guess this has to wait until http://bugs.winehq.org/show_bug.cgi?id=25060
is fixed, or until someone finds a better workaround.
```

Original comment by `daniel.r...@gmail.com` on 21 Jun 2012 at 5:14
- Changed state: **Upstream**

**Comment by GoogleCodeExporter** [comment]
```
"winetriccks -q dotnet35" still hangs for me at the end, but
if I then do "winetricks dotnet_verifier; wine netfx_setupverifier.exe",
and verify the dotnet35 installation, it passes.  So maybe the resulting 
installation
is really ok?
```

Original comment by `daniel.r...@gmail.com` on 5 Jul 2012 at 1:49

**Comment by GoogleCodeExporter** [comment]
```
I think this is fixed now in current cvs with wine-1.6rc3.  Please retest.
```

Original comment by `daniel.r...@gmail.com` on 6 Jul 2013 at 5:53
- Changed state: **Fixed**
